doc/endpoint/schemas.md

```diff
@@ -117,6 +117,23 @@ val s: Schema[StringOrInt] = Schema.derivedUnion[StringOrInt]
 If any of the components of the union type is a generic type, any of its validations will be skipped when validating
 the union type, as it's not possible to generate a runtime check for the generic type.
 
+### Derivation for string-based constant union types
```

Comment:
thanks for adding the docs, maybe they should go to https://tapir.softwaremill.com/en/latest/endpoint/enumerations.html though?